src/garage/torch/policies/discrete_policy.py

```diff
@@ -0,0 +1,70 @@
+"""Base Discrete Policy."""
+import abc
+
+import akro
+import torch
+
+from garage.torch import global_device
+from garage.torch.policies.policy import Policy
+
+
+class DiscretePolicy(Policy, abc.ABC):
```

Comment:
hmm is there a reason we need a DiscretePolicy API? how is this different from a policy?

Reply:
Given the contents of the methods here, I think this should actually inherit from `StochasticPolicy`.

Reply:
I definitely think this should just be `StochasticPolicy`, the dictionary should just be empty.